src/prog_models/models/centrifugal_pump.py

```diff
@@ -121,79 +122,92 @@ class CentrifugalPump(prognostics_model.PrognosticsModel):
             'To': 350
         },
 
+        # Wear Rates
+        'wA': 0,
+        'wRadial': 0,
+        'wThrust': 0,
+
         # Initial state
         'x0': {
-            'w': 376.991118431, # 3600 rpm (rad/sec)
+            'w': 376.991118431,  # 3600 rpm (rad/sec)
             'Q': 0,
             'Tt': 290,
             'Tr': 290,
             'To': 290,
             'A': 12.7084,
             'rThrust': 1.4e-6,
             'rRadial': 1.8e-6,
-            'wA': 0.0,
-            'wThrust': 0,
-            'wRadial': 0,
         }
     }
 
+    state_limits = {
+        'To': (-273.15, inf),
+        'Tr': (-273.15, inf),
+        'Tt': (-273.15, inf),
+        'A': (0, inf)
+    }
+
     def initialize(self, u, z = None):
         x0 = self.parameters['x0']
-        x0['QLeak'] = math.copysign(self.parameters['cLeak']*self.parameters['ALeak']*math.sqrt(abs(u['psuc']-u['pdisch'])), u['psuc']-u['pdisch'])
+        x0['QLeak'] = math.copysign(\
+            self.parameters['cLeak']*self.parameters['ALeak']*\
+                math.sqrt(abs(u['psuc']-u['pdisch'])), u['psuc']-u['pdisch'])
```

Comment:
From the following paper, it seems like the sign should be reversed (i.e. u['pdisch']-u['psuc'] instead of u['psuc']-u['pdisch']):
Daigle, M. J., & Goebel, K. (2013). Model-based prognostics with concurrent damage progression processes. IEEE Transactions on Systems, Man, and Cybernetics: Systems, 43(3), 535–546. https://doi.org/10.1109/tsmca.2012.2207109

Reply:
Currently investigating this one. The implemented solution is consistent with the matlab implementation as well. Currently I suspect this might be a mistake in the paper, but we're working to confirm.